### ISSUE TITLE
tmt plans lint for unknown attributes

### ISSUE THREAD

**Issue by lukaszachy** [psss/tmt#707]
Inspiration in pull/639.
For example - currently lint exists with 0 even though step discoverAAA is clearly typo / not valid.
```
summary: Essential command line features
execute:
    how: tmt
discoverAAAA:
    how: fmf
    url: https://github.com/psss/tmt
```

**Comment by adka1408** [comment]
I'll take a look at that.

**Comment by FrNecas** [comment]
Fixed by #809
